frontend/pages/api/search.ts

```diff
@@ -0,0 +1,21 @@
+import MeiliSearch from 'meilisearch';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { MemberHit } from '~/types/MemberHit';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (req.method !== 'GET') {
+    res.status(404);
+  }
+
+  const searchQuery = req.query.q as string;
+  const escapedQuery = searchQuery.replace(/[-[\]{}()*+?.,\\^$|#\s]/g, '\\$&');
+
+  const client = new MeiliSearch({
```

Comment:
It should be possible to create the MeiliSearch client outside of the handler, so on row 4. That way it gets created when the server is started and doesn't have to be created again on every request. I'm not sure if that's a good idea/best practice though.